### PR TITLE
Audio Player/Reader Project

### PR DESCRIPTION
**Pull request by Ameboid**
## Submission Checklist:

- [X] I am a current high school, middle school, or home schooled student.
  - [X] I am 18 or under
  - [X] I have filled out the [verification form](https://verify.onboard.hackclub.com) **(do this for every pull request)**
    -  *or the [legacy verification form](https://legacy.verify.onboard.hackclub.com) if the above doesn't work* 
- [X] I have joined the [`#electronics`](https://hackclub.slack.com/archives/C056AMWSFKJ) channel on [Slack](https://hackclub.com/slack/?event=onboard)
- [X] I made this board from scratch, even if I followed a tutorial
- [X] I have followed the directions in [README.md](https://github.com/hackclub/OnBoard/blob/main/README.md#getting-started)
  - [X] Created a folder under `onboard/projects`
  - [X] Filled out `TEMPLATE.md` as `README.md` inside your project folder
  - [X] Uploaded `gerber.zip` and `schematic.pdf` inside your project folder
  - [X] Created a folder called `src` and uploaded design files
  - [X] Uploaded screenshot of PCB order called `cart.png` with all costs included
  - [X] I am ordering the smallest batch size (ie. JLCPCB's smallest order is for 5 boards)
  - [ ] If outside the US I've checked that I can afford the customs charges in my country (which isn't covered by the grant)

- [ ] (Optional) This project is from a tutorial[^1].
- [ ] (Optional) This project is for OnBoard Live[^2].
- [ ] (Optional) This project is for Hacky Holidays[^3].
- [X] (Optional) I'm in a FIRST (FRC, FTC, FLL, etc.) team. The number is: 6421

[^1]: Projects from a tutorial are 100% fine! We just want to ask so we can count how people are using tutorials.
[^2]: OnBoard Live is a special version of OnBoard where you can earn more money for designing advanced boards. Check out the #onboard-live channel on [our Slack](https://hackclub.com/slack/?event=onboard)!
[^3]: [Hacky Holidays](https://hacky-holidays.hackclub.com/) is a limited time PCB event, design a holiday-themed PCB this Winter, and get it shipped.

### CODE REVIEW

**Comment by prophetorpheus** [comment]
Hi, I'm Orpheus Leap! Here to help you review your PR.
  
**projects/ESP32_Audio_Reader__Player_Card:**

Required files
||||
|---|---|---|
|✅| README.md | A description of your project |
|✅ | cart.png |  |
**projects/ESP32_Audio_Reader__Player_Card/gerber.zip:**

	
  Happy OnBoarding!
 <sub>This bot is a simpler helper for common submission types. If there's an error with this, please point it out and someone from the OnBoard team will look at it manually.</sub>

**Comment by Ameboid** [comment]
I also wanted to ask about ordering parts from LCSC, since the PCBA would exceed the grant, and I have the knowledge and tools to assemble the board myself.

**Review by LimesKey — CHANGES_REQUESTED** [review]
Thanks for submitting your PCB pull request for OnBoard! However, it looks like you took a picture of your order details rather than the completed unpaid order.  For us to have the full picture of pricing and the PCB your ordering, please follow the steps in this guide [here](https://github.com/hackclub/OnBoard/blob/main/docs/ordering_from_JLCPCB.md) or here:

1. Click "Secure Checkout": After adding your PCB to the cart
2. Enter your shipping information and add a coupon code if you have one, then click "Submit Order", do not add any payment options
3. Close that tab and open a new one to the [JLCPCB Order page](https://jlcpcb.com/user-center/orders/), then take a rectangular screenshot of the PCB you just submitted. It should show all the details of your order including the price and assembly.

This will allow us to spot any mistakes and give you ways to reduce the cost or customs fee. Congratulations on making the PCB! Please ask in [#electronics](https://slack.com/archives/C056AMWSFKJ) in the [Slack](https://hackclub.com/slack/) if you need any further help.

**Comment by Ameboid** [comment]
I would like to ask if it is possible to include the ordering of parts for
my project, and if anything special should be done.

Thank you!

**Comment by WolfGamer2** [comment]
Heya! You can get up to 5 pcb's with the grant. Please follow the instructions above so we can move forward!

**Comment by Ameboid** [comment]
Oh, I know that part. I was asking, since the PCBA is over the 100 of the
grant. Ordering the PCB's themselves is a minuscule amount, and I didn't
want to have to order the boards using a fraction of the total cost, only
to pay for the components and have to assemble it myself. Because I have
the resources necessary to make the board itself.

**Comment by WolfGamer2** [comment]
Could you rephrase what you are asking? Are you saying you only want a part of the pcb and the rest in parts?

**Comment by Ameboid** [comment]
That is pretty much it. I would like the PCB's to be manufactured, and I
will assemble two of them with the ordered parts. A stencil for laying the
solder paste is included in the order as well.

**Comment by WolfGamer2** [comment]
As long as you stay under the grant its fine!

**Comment by Ameboid** [comment]
Got it!

**Comment by WolfGamer2** [comment]
Have you updated the image? Once that is done I can issue the grant.

**Comment by Ameboid** [comment]
No. But should I have two separate images for the PCB and the parts? Since
I belive that they are separate parts of the JLC ordering service. If not,
then I'll see what I can do to view all orders.

**Comment by WolfGamer2** [comment]
Yep, just send the actual pcb screenshot as cart.png and send the other one in the chat.

**Comment by Ameboid** [comment]
So there are three backordered items that do not have any substitutes
available. Should I just include the image of the in-stock?

I am also using the LCSC Ordering Service since JLC Parts order is only for
PCBA

**Comment by WolfGamer2** [comment]
As long as you stay in the grant it is fine.

**Comment by WolfGamer2** [comment]
I will send the grant tmrw!

**Comment by Ameboid** [comment]
Well it's almost tomorrow...

**Comment by WolfGamer2** [comment]
Pretty late for me, tmrw morning.

**Comment by Ameboid** [comment]
So for the BOM I have it included in the PR files. Does it require a
screenshot of the order from LCSC?

**Comment by WolfGamer2** [comment]
Yes, it does.

**Comment by Ameboid** [comment]
Got it! Getting right on it.

**Comment by Ameboid** [comment]
Right now getting the quote for the parts from LCSC.

**Comment by WolfGamer2** [comment]
Ok, thanks for the update.

**Comment by Ameboid** [comment]
Back to the Order, should it be the Checkout information or the Quote
information?

Included is the Quote information and the Order Details

**Comment by WolfGamer2** [comment]
Heya! I cannot see the images, can you please upload them?

**Comment by Ameboid** [comment]
Really? Are they too blurry, or are they not viewable?

**Comment by WolfGamer2** [comment]
I don't see where you put them. Can you upload and let me know.

**Comment by Ameboid** [comment]
Here they are, I think. If they are not viewable, then I'll try something
else.

**Comment by LimesKey** [comment]
The files you uploaded were corrupt and aren't able to be opened, try adding them again.

**Comment by WolfGamer2** [comment]
Yup!

**Comment by Ameboid** [comment]
Wut?

**Comment by LimesKey** [comment]
Upload the new files please! Ask in #electronics in the Slack if you need help.

**Comment by Ameboid** [comment]
Do you want me to upload the pictures of the payment information or of the
parts?

**Comment by LimesKey** [comment]
Looks good, but you do understand that OnBoard can only pay for your JLCPCB order? Thanks for the LCSC BOM though

**Comment by Ameboid** [comment]
Aww nuts. Oh well.

**Comment by LimesKey** [comment]
I can't seem to find your verification form, are you sure you submitted it?

**Comment by Ameboid** [comment]
Verification form?

**Comment by LimesKey** [comment]
Here, https://verify.onboard.hackclub.com/

**Comment by Ameboid** [comment]
I can sign into GitHub just fine, but when I try to do the slack,
wellll.......

**Comment by WolfGamer2** [comment]
Try again!

**Comment by Ameboid** [comment]
This time it showed that I was logged into gothub, but it still told me
that I needed to, and I think, install it on the Hack Club slack?

**Comment by WolfGamer2** [comment]
yeah!

**Comment by Ameboid** [comment]
It asked me to install OnBoard Verifier on the Hack Club slack, saying that
I don't have permission to, and it does it consistently

**Comment by WolfGamer2** [comment]
@LimesKey

**Comment by Ameboid** [comment]
Has something gone wrong? Do I need to remerge the project?

**Comment by maxwofford** [comment]
@Ameboid hey– just a heads up that you need to be a full member of the hack club slack to apply for onboard. seems like you're still a multi-channel guest. I can promote you to a full user if you post your slack username here & get that verification working for you.

**Comment by Ameboid** [comment]
Ameboid, like that. If that's not it, then I don't know what else.

**Comment by maxwofford** [comment]
I just promoted you to a full user @Ameboid – try the form again!

**Comment by Ameboid** [comment]
It works!

**Comment by Ameboid** [comment]
Well, now it just says "Loading..." for ages...

**Comment by maxwofford** [comment]
@Ameboid do you have an ad blocker? can you try incognito?

**Comment by Ameboid** [comment]
Okay- now it worked

**Comment by maxwofford** [comment]
Got it! Nice work– I'll get it verified a little later today & get that grant sent out.

**Comment by maxwofford** [comment]
Nice work on all this @Ameboid! I just sent out a grant you can use to fab the board: https://hcb.hackclub.com/grants/zxIEEJ
